File: attic/xcode/ServerConfigTests/ServerConfigTests.m
//
//  ServerConfigTests.m
//  ServerConfigTests
//
//  Created by Dirk Zimmermann on 17/01/2017.
//  Copyright © 2017 pEp Security S.A. All rights reserved.
//

#import <XCTest/XCTest.h>

#import "ServerConfig.h"

@interface ServerConfigTests : XCTestCase

@end

@implementation ServerConfigTests

- (void)testSimpleLookup {
    SCAccount *ac = [[SCAccount alloc] init];
    ac.email = @"someguy@gmail.com";
    NSInteger res = [ac probe];
    XCTAssertEqual(res, 0);
}

@end

File: attic/xcode/ServerConfigTests/ServerConfigTests.swift
//
//  ServerConfigTests.swift
//  ServerConfig
//
//  Created by Dirk Zimmermann on 18/01/2017.
//  Copyright © 2017 pEp Security S.A. All rights reserved.
//

import XCTest

class ServerConfigTests: XCTestCase {

    func testSimpleLookup() {
        let ac = SCAccount()
        ac.email = "someone@gmail.com"
        let res = ac.probe()
        XCTAssertEqual(res, 0)
    }
}
